### PR TITLE
Links glossary

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/glossary/a.md [hunk 1]
```diff
@@ -2,46 +2,40 @@
...
 
-<a name="administration-app"></a>
-
 ### Administration application
 
 The Administration application is one of the default applications of {{< product-c8y-iot >}}.
 The Administration application enables administrators to manage their tenants, users, roles and applications. It also allows them to configure various settings for their accounts.
 
-For details see [Administration](/users-guide/administration) in the *User guide*.
+For details see [Standard tenant administration](/standard-tenant/).
```
Comment (rssag) [inline content/glossary/a.md:14]: just a thought - would full paths be useful in the glossary as well? similar reasoning to what we decided for related topics. maybe not for glossary-internal links, but links to other parts of the documentation.

**Review by rssag — APPROVED** [review]
one thought + approval

File: content/glossary/a.md (hunk 1 above)
Comment (MWindel) [reply content/glossary/a.md:14]: I agree. The glossary is in some way similar to related topics. A full link would be beneficial here.

File: content/glossary/a.md [hunk 2]
```diff
@@ -2,61 +2,53 @@
...
 
-<a name="administration-app"></a>
-
 ### Administration application
 
 The Administration application is one of the default applications of {{< product-c8y-iot >}}.
 The Administration application enables administrators to manage their tenants, users, roles and applications. It also allows them to configure various settings for their accounts.
 
-For details see [Administration](/users-guide/administration) in the *User guide*.
+For details see [Platform administraion > Standard tenant administration](/standard-tenant/).
```
Comment (rssag) [inline content/glossary/a.md:14]:
```suggestion
For details see [Platform administration > Standard tenant administration](/standard-tenant/).
```

File: content/glossary/s.md [hunk 3]
```diff
@@ -38,4 +38,4 @@ The application concept of {{< product-c8y-iot >}} includes a basic application
 Tenants can be subscribed to applications which have been deployed by their superior tenant ({{< management-tenant >}} or {{< enterprise-tenant >}}).
 Granting access to subtenants and subscribing to applications is done in the Administration application.
 
-For details see [Application subscription](/concepts/applications/#subscription) and [Administration > Managing applications](/standard-tenant/ecosystem/#managing-applications).
+For details see [Getting started > Technical concepts > Developing applications > Application subscription](/concepts/applications/#subscription) and [Platform administration > Standard tenant administration > Managing the ecosystem > Managing applications](/standard-tenant/ecosystem/#managing-applications).
```
Comment (MWindel) [inline content/glossary/s.md:41]: How do we go about variables here?

File: content/glossary/s.md [hunk 4]
```diff
@@ -2,43 +2,40 @@
...
-<a name="subscription"></a>
 
 ### Subscription
 
 The application concept of {{< product-c8y-iot >}} includes a basic application marketplace.
 Tenants can be subscribed to applications which have been deployed by their superior tenant ({{< management-tenant >}} or {{< enterprise-tenant >}}).
 Granting access to subtenants and subscribing to applications is done in the Administration application.
 
-For details see [Developing applications > Subscribing applications](/concepts/applications/#subscription) and [Administration > Managing applications](/users-guide/administration/#managing-applications) in the *User guide*.
+For details see [Getting started > Technical concepts > Developing applications > Application subscription](/concepts/applications/#subscription) and [Platform administration > Standard tenant administration > Managing the ecosystem > Managing applications](/standard-tenant/ecosystem/#managing-applications).
```
Comment (BeateRixen) [inline content/glossary/s.md:41]:
```suggestion
For details see [Getting started > Technical concepts > Developing applications > Application subscription](/concepts/applications/#application-subscription) and [Platform administration > {{< standard-tenant >}} administration > Managing the ecosystem > Managing applications](/standard-tenant/ecosystem/#managing-applications).
```

File: content/glossary/s.md (hunk 3 above)
Comment (BeateRixen) [reply content/glossary/s.md]: added

**Pull request merged**